### PR TITLE
fixbug helm 内 clusterrole scheduler 缺少 verbs

### PR DESCRIPTION
**Pull request by weapons97**
/assign @hzxuzhonghu

### CODE REVIEW

File: installer/helm/chart/volcano/templates/scheduler.yaml [hunk 1]
```diff
@@ -78,7 +78,7 @@ rules:
     verbs: ["get", "list", "watch", "delete"]
   - apiGroups: [""]
     resources: ["configmaps"]
-    verbs: ["get", "create", "delete", "update"]
+    verbs: ["get", "create", "delete", "update", "list", "watch"]
```
Comment (hzxuzhonghu) [inline installer/helm/chart/volcano/templates/scheduler.yaml:81]: @wangyang0616 I donot see where scheduler list-watch configmaps

**Comment by weapons97** [comment]
> I think it is reasonable to add the watch and list permissions of configmap. Can you describe the problems encountered during the current operation in the PR?

**Comment by weapons97** [comment]
> I think it is reasonable to add the watch and list permissions of configmap. Can you describe the problems encountered during the current operation in the PR?

如果缺少 list watch 字段，不知为何，会导致npu 的vcjob 一直持续在pending 状态。scheduler 报错是。
Add enqueue failed, require npu num is 1 but cluster npu num is 0
image
对比 helm charts，跟 volcano-v1.7.0.yaml 的部署文件，发下缺少的就是这个权限。

File: installer/helm/chart/volcano/templates/scheduler.yaml (hunk 1 above)
Comment (weapons97) [reply installer/helm/chart/volcano/templates/scheduler.yaml:81]: image
不加这个权限，scheduler 会报错。

Comment (wangyang0616) [reply installer/helm/chart/volcano/templates/scheduler.yaml:81]: I just verified locally that there is no such error message when helm installs Volcano 1.8.1. I also confirmed the Volcano code. Currently, there is no list watch operation on the configmap.

Please provide the K8s and Volcano versions you are using. Is there any additional development based on Volcano?

**Comment by wangyang0616** [comment]
> > I think it is reasonable to add the watch and list permissions of configmap. Can you describe the problems encountered during the current operation in the PR?
> 
> 如果缺少 list watch 字段，不知为何，会导致npu 的vcjob 一直持续在pending 状态。scheduler 报错是。 Add enqueue failed, require npu num is 1 but cluster npu num is 0 image 对比 helm charts，跟 volcano-v1.7.0.yaml 的部署文件，发现缺少的就是这个权限。

According to the description in your log, NPU resources are used. Could you please provide your NPU card model information? As far as I know, community Volcano currently does not support the scheduling capabilities of the Ascend NPU D910 series.

**Comment by weapons97** [comment]
> > > I think it is reasonable to add the watch and list permissions of configmap. Can you describe the problems encountered during the current operation in the PR?
> > 
> > 
> > 如果缺少 list watch 字段，不知为何，会导致npu 的vcjob 一直持续在pending 状态。scheduler 报错是。 Add enqueue failed, require npu num is 1 but cluster npu num is 0 image 对比 helm charts，跟 volcano-v1.7.0.yaml 的部署文件，发现缺少的就是这个权限。
> 
> According to the description in your log, NPU resources are used. Could you please provide your NPU card model information? As far as I know, community Volcano currently does not support the scheduling capabilities of the Ascend NPU D910 series.

volcano 是加了ascend-device-plugin 编译部署的。如果加了这2个verb 也能正常调度npu，npu型号是310P。

File: installer/helm/chart/volcano/templates/scheduler.yaml (hunk 1 above)
Comment (weapons97) [reply installer/helm/chart/volcano/templates/scheduler.yaml:81]: k8s: 1.24
volcano: 1.7.0 + ascend-volcano-plugin v5.0.0-RC3.2
在 ascend-volcano-plugin/build 目录下的volcano-v1.7.0.yaml ，scheduler ClusterRole是有configmap list watch 的权限的，我猜是ascend-volcano-plugin 那边需要这个权限？

Comment (wangyang0616) [reply installer/helm/chart/volcano/templates/scheduler.yaml:81]: Does ascend-volcano-plugin use the main Volcano branch, or does it maintain a separate Volcano branch?

Comment (weapons97) [reply installer/helm/chart/volcano/templates/scheduler.yaml:81]: 用的是  v5.0.0-RC3.2 这个tag

**Pull request closed without merging**